cargo-dist/tests/integration-tests.rs

```diff
@@ -62,6 +62,42 @@ path-guid = "BFD25009-65A4-4D1E-97F1-0030465D90D6"
     })
 }
 
+#[test]
+fn axolotlsay_custom_formula() -> Result<(), miette::Report> {
+    let test_name = _function_name!();
+    AXOLOTLSAY.run_test(|ctx| {
+        let dist_version = ctx.tools.cargo_dist.version().unwrap();
+        ctx.patch_cargo_toml(format!(r#"
+[workspace.metadata.dist]
+cargo-dist-version = "{dist_version}"
+installers = ["homebrew"]
+tap = "axodotdev/homebrew-packages"
+formula = "AXOLOTLSHOUT"
```

Comment:
We probably want to test with a lowercase name, not an uppercase name. Capitalization is significant and has some implications.

We could also consider normalizing the name based on other factors in a later update, but I think it also just makes sense to link people to the Homebrew docs about formula names when they're picking one.